.github/workflows/pull_requests.yml

```
@@ -54,8 +54,8 @@ jobs:
           pr-message: |-
             Welcome @${{github.actor}} :wave:
 
-            It looks like this is your first Pull Request submission to the [Terraform AWS Provider](https://github.com/hashicorp/terraform-provider-aws)! If you haven’t already done so please make sure you have checked out our [CONTRIBUTING](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing) guide and [FAQ](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/faq.md) to make sure your contribution is adhering to best practice and has all the necessary elements in place for a successful approval.
+            It looks like this is your first Pull Request submission to the [Terraform AWS Provider](https://github.com/hashicorp/terraform-provider-aws)! If you haven’t already done so please make sure you have checked out our [CONTRIBUTING](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/index.md) guide and [FAQ](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/faq.md) to make sure your contribution is adhering to best practice and has all the necessary elements in place for a successful approval.
 
-            Also take a look at our [FAQ](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/faq.md) which details how we prioritize Pull Requests for inclusion.
+            Also take a look at our [FAQ](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/faq.md) which details how we prioritize Pull Requests for inclusion.
```

We have a full writeup on prioritization now, so linking that instead.

```suggestion
            Also take a look at our [prioritization guide](https://hashicorp.github.io/terraform-provider-aws/prioritization/) which details how we prioritize Pull Requests for inclusion.
```